i18n/PRINCIPLES_pt.md

```diff
@@ -5,18 +5,18 @@ Esses princípios são derivados de operações de software modernas, mas també
 
 O [estado desejado](./GLOSSARY_pt.md#estado-desejado) de um sistema gerenciado por GitOps deve ser:
 
-1. **Declarativo**
+1. ## Declarativo
 
     Um [sistema](./GLOSSARY_pt.md#sistema-de-software) onde seu estado desejado é expresso [declarativamente](./GLOSSARY_pt.md#descrição-declarativa).
 
-2. **Versionado e Imutável**
+2. ## Versionado e Imutável
 
     O estado desejado é [armazenado](./GLOSSARY_pt.md#armazenamento-de-estado) de uma forma que impõe imutabilidade, controle de versão e retém um histórico de versão completo.
 
-3. **Obtido Automaticamente**
+3. ## Obtido Automaticamente
```

Comment:
Extraido ao invés de obtido

Reply:
I would like to understand the justification for using 'extraído' instead of 'obtido' here. And perhaps 'puxado' would be an even better fit here because we're using the verb 'puxar' in the glossary entry for "Pull" (see above).

Reply:
Extraido might suggest the content was removed from source so Puxado makes more sense